packages/blockchain-wallet-v4-frontend/src/data/components/send/selectors.ts

```diff
@@ -9,3 +9,11 @@ export const getPaymentsAccountExchange = curry(
     )
   }
 )
+
+export const getPaymentsTradingAccountAddress = curry(
+  (currency: CoinType, state: RootState) => {
+    return state.components.send.tradingPaymentsAccount[currency].map(
+      x => x.address
+    ).data
```

Comment:
`.data` should **NEVER** be used, _unless_ you know **FOR SURE** the data is **Success**ful